images/linux/scripts/tests/Common.Tests.ps1

```diff
@@ -0,0 +1,30 @@
+Describe "PHP" {
+
+    [array]$testCases = (Get-ToolsetContent).php.versions | ForEach-Object { @{phpVersion = $_} }
+
+    It "php <phpVersion>" -TestCases $testCases {
+        param (
+            [string] $phpVersion
+        )
+
+        "php$phpVersion --version" | Should -ReturnZeroExitCode
+        "php-config$phpVersion --version" | Should -ReturnZeroExitCode
+        "phpize$phpVersion --version" | Should -ReturnZeroExitCode
+    }
+
+    It "PHP-Unit" {
```

Comment:
correct naming is `PHPUnit` https://phpunit.de/

Reply:
Fixed, thank you!